src/app/MessageDef/SubscribeRequest.cpp

```diff
@@ -197,12 +197,12 @@ CHIP_ERROR SubscribeRequest::Parser::GetEventNumber(uint64_t * const apEventNumb
 
 CHIP_ERROR SubscribeRequest::Parser::GetMinIntervalSeconds(uint16_t * const apMinIntervalSeconds) const
 {
-    return GetUnsignedInteger(kCsTag_EventNumber, apMinIntervalSeconds);
+    return GetUnsignedInteger(kCsTag_MinInterval, apMinIntervalSeconds);
```

Comment:
kCsTag_MinIntervalSec or Seconds please. Not having suffixes for tags/values causes bugs.

Reply:
Done, thanks